wfl/cli/cli_options.py

```diff
@@ -1,12 +1,52 @@
-"""
-These are generalised cli options, merged decorators for repeated operations
-"""
+import click
+from wfl.configset import ConfigSet, OutputSpec
+from ase.io.extxyz import key_val_str_to_dict
 
-from click import STRING, argument, option
 
+def _to_ConfigSet(ctx, param, value):
+    return ConfigSet(value)
 
-def file_input_options(f):
-    """"""
-    f = argument("inputs", nargs=-1)(f)
-    f = option("--index", "-i", type=STRING, required=False, help="Pass this index to configset globally")(f)
+def inputs(f):
+    """Create ConfigSet for given filename(s)"""
```

Comment:
This is the docstring for this decorator, right? How about 
```
"""add a standard option for the ConfigSet inputs"""
````
and same for `outputs()`.